### DISCUSSION TITLE
Why the spy jar uses such a generic file name, instead of pgjdbc-ng-spy-<version>?

### DISCUSSION THREAD

**Discussion by butlnor** [impossibl/pgjdbc-ng#526] [General]
The title says it all. There can be many different libraries, that uses "spy" as the name, so this can clearly and unnecessarily lead to the "jar hell".

Please rename jar into a more intuitive pgjdbc-ng-spy-&lt;version&gt;.jar

**Comment by kdubb** [comment]
@butlnor Fairly good point, this just happens to be the default naming from Gradle.

**Comment by kdubb** [comment]
I've updated the build so any release _after_ the current release (e.g. > 0.8.7), all the published jars will be prefixed with `pgjdbc-ng-`.
